nyaa/views/account.py

```diff
@@ -223,6 +223,48 @@ def profile():
     return flask.render_template('profile.html', form=form)
 
 
+@bp.route('/trusted/request', methods=['GET', 'POST'])
+def request_trusted():
+    if not flask.g.user:
+        return flask.redirect(flask.url_for('account.login'))
+    trusted_form = None
+    deny_reasons = []
+    if flask.request.method == 'POST':
+        trusted_form = forms.TrustedForm(flask.request.form)
+        if trusted_form.validate():
```

Comment:
`deny_reasons` is never computed/checked for POST requests, so if you don't pass minimum requirements, you can just directly POST the data.

Reply:
They would have a hard time getting past trusted_form.validate() without a csrf token which they never get because they never get the form, but sure, I've changed to code so that it checks the `deny_reasons` both paths.